ansible/roles/pacmod/tasks/main.yaml

```diff
@@ -1,3 +1,11 @@
+- name: Install plotjuggler
+  become: true
+  ansible.builtin.apt:
+    name:
+      - ros-{{ rosdistro }}-plotjuggler-ros
```

Comment:
Why is plotjuggler being installed in the pacmod role?

We should either move this to somewhere else or rename this role.

It is also not mentioned in the readme file.